keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -3117,6 +2966,39 @@ already received all the relevant updates (Pod, ResourceClaim, etc.).
   - Testing: An E2E test covers the expected retry mechanism in kubelet when
     `NodePrepareResources` fails intermittently.
 
+- A DRA driver prepares a ResourceClaim for a pod, then fails or gets uninstalled.
```

Comment:
should we add the pod scheduling timeouts here as a failure mode?

Reply:
Added.